prov/verbs/src/verbs_msg_ep.c

```diff
@@ -170,28 +170,41 @@ static int fi_ibv_msg_ep_enable(struct fid_ep *ep)
 {
 	struct ibv_qp_init_attr attr;
 	struct fi_ibv_msg_ep *_ep;
+	struct ibv_pd *pd;
 
 	_ep = container_of(ep, struct fi_ibv_msg_ep, ep_fid);
 	if (!_ep->eq)
 		return -FI_ENOEQ;
-	if (!_ep->scq || !_ep->rcq)
+	if (!_ep->scq && !_ep->rcq)
```

Comment:
Can this check be expanded to include the tx/rx sizes?

if ((!_ep->scq && _ep->info->tx_attr->size) || (!_ep->rcq && _ep->info->rx_attr->size))